src/views/UserProfileView.vue

```diff
@@ -443,7 +441,7 @@ export default {
 </script>
 
 
-<style lang="scss" scoped>
+<style lang="scss">
```

Comment:
Looks good to me. I'm not really sure if we should remove `scoped` here or add the style to the Avatar component.

cc @venimus what do you think?

Reply:
Imho the style might be better in the parent component, which can set the effects /sizes and etc required in its context, so I think scoped should probably stay, tough i don't seem much problem as the styling in this case is only for a component that appears only 1 on the page (also the style selectors are css-scoped already - not too generic)

Reply:
I saw that class .description in User Profile Page is breaking the description in preview of a tip. If we are removing "scoped", please add wrapper class for those pages. For example all styles for User Profile page should go under .profile__page .